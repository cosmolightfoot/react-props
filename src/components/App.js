import React from 'react';
import Color from './Color.js';

export default function App() {
  return (
  <>
    <Color name="black" hex="000000" rgb={{ red: 0, green: 0, blue: 0 }} />
  </>
  );
}
