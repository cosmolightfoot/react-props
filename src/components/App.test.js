import React from 'react';
import { shallow } from 'enzyme';
import App from './App.js';

describe('App component test', () => {
  it('renders App', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
