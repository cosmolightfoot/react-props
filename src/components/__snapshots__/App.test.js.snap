// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component test renders App 1`] = `
<Fragment>
  <ColorPicker />
</Fragment>
`;
